keps/0000-kep-template.md

```diff
@@ -86,9 +83,24 @@ A table of contents is helpful for quickly jumping to sections of a KEP and for
 
 [Tools for generating]: https://github.com/ekalinin/github-markdown-toc
 
+**ACTION REQUIRED:** There must be an issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) referencing this KEP and targeting a release milestone **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)
```

Comment:
This needs a reason clause attached - “in order to merge code into a given release, there must be an issue in ...”

Otherwise it reads like “must do things for no reason”

Reply:
Honestly - I think this should be at the top around 43 since it's a checklist.

```
1. File an issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) referencing this KEP and targeting a release milestone. This KEP must be approved and marked `implementable` **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)** of that milestone. Likewise, when the enhancement is ready to be promoted between alpha/beta/stable - file a new enhancement in that milestone to track it.

Reply:
This assumes the KEP is for something that will be parts of the k8s/k8s release package. Some things may not fall along those release milestones (e.g., an enhancement to kubebuilder or minikube).

This would only apply to things that are on the core k8s/k8s release process. How do we capture that?

Reply:
It's kind of circular, but maybe say it needs to be done for anything to be tracked as part of a Kubernetes release? That may include things not in kubernetes/kubernetes (e.g., release-specific docs) and things included only by reference in the release bundle, such as a number of container images built from other repos. But it would exclude things tied to Kubernetes releases but that aren't part of the release process, such as client-go and minikube.